packages/hydrooj/src/service/server.ts

```diff
@@ -606,6 +605,7 @@ ${ctx.response.status} ${endTime - startTime}ms ${ctx.response.length}`);
                 socket.close(1003, 'Websocket Error');
             } catch (e) { }
         });
+        socket.pause();
```

Comment:
At L455. Pausing the event before any await function calls and resuming the event after on message handle installed to avoid ignorance of any message.